kafka/src/main/java/io/micronaut/configuration/kafka/processor/ConsumerInfo.java

```diff
@@ -72,6 +72,9 @@ final class ConsumerInfo {
     final boolean trackPartitions;
     final boolean shouldSendOffsetsToTransaction;
 
+    private final Map<String, Argument<?>> seekArgCache = new HashMap<>();
+    private final Map<String, Argument<?>> ackArgCache = new HashMap<>();
```

Comment:
Is concurrent access to these hash maps possible? Should they be concurrent implementations?

Reply:
No concurrent access possible 👇 

```java
final KafkaSeekOperations seek = Optional.ofNullable(info.seekArg(consumerRecord.topic())).map(x -> KafkaSeekOperations.newInstance()).orElse(null);
            Optional.ofNullable(info.seekArg(consumerRecord.topic())).ifPresent(argument -> boundArguments.put(argument, seek));
```

Do you think it is worth it to instantiate it as ConcurrentHashMap just to be safe for the future?